packages/code-connect/components/Banner/NonStatusBanner.figma.tsx

```diff
@@ -0,0 +1,39 @@
+import { Banner } from '@patternfly/react-core';
+import figma from '@figma/code-connect';
+
+/**
+ * PatternFly Banner integration for Figma Code Connect
+ */
+
+figma.connect(
+  Banner,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6%3A-Components-Test?node-id=221-1443&m=dev',
+  {
+    props: {
+      text: figma.string('✏️ Center text'),
+
+      // enum
+      color: figma.enum('Color', {
```

Comment:
valid colors are:
'red' | 'orangered' | 'orange' | 'yellow' | 'green' | 'teal' | 'blue' | 'purple'

Reply:
color="yellow" and color="teal" are missing